### PR TITLE
Cross platform post-install

### PR DESCRIPTION
**Pull request by hugihlynsson**
Fixes the `; true` part tripping up the post install step on some platforms by moving the error silencing logic into the cli code.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
       - name: Install modules
-        run: yarn install --immutable
+        run: yarn install --immutable --ignore-scripts
```
Comment (bragur) [inline .github/workflows/ci.yml:19]: Hmm, seems like `--immutable` is supported in v2 only and `--ignore-scripts` only in v1 🤔

Comment (hugihlynsson) [reply .github/workflows/ci.yml:19]: Ohh! We're using version 1 here, seeing that this change had the desired impact. This was renamed from `--frozen-lockfile`, changing it to that!

Comment (hugihlynsson) [reply .github/workflows/ci.yml]: Fixed!

**Review by bragur — APPROVED** [review]
LGTM!

**Pull request merged**